### ISSUE TITLE
Cannot scroll to last item if has many items

### ISSUE THREAD

**Issue by khanitnanj** [davidcetinkaya/embla-carousel#64]
first, thanks for you library it's very helpful for my work.

I have some issue when the container has lot of items. ( Ex. 50 items with 150px of width) 

I look in the element inspector 

this code 
`transform: translate3d(-1805.47%, 0px, 0px)`

I guess it's always back to -1800% 

how to increase?

**Comment by davidcetinkaya** [comment]
Hello @khanitnanj,

Please provide a test case like a [CodeSandbox](https://codesandbox.io/) or [CodePen](http://codepen.io/) that clearly demonstrates the issue. I need this to understand the issue you're experiencing and in order to debug it.

Kindly,
David

**Comment by khanitnanj** [comment]
Hi @davidcetinkaya 

I've created a CodeSandbox
https://codesandbox.io/s/embla-carousel-react-basic-setup-0ky1p

This is what I found, but I have fixed on my project

The problem is 
 
1.  I set width for `.content`  to small box you can try at 300px
2. if `.embla__slide` I set the item width to 150px (flex: 0 0 150px)
3. I have items 50 items

it's will cannot scroll to the last item

but I fixed by changing some elements and and now on my project fixed. 

Thank you for your super fast reply

**Comment by davidcetinkaya** [comment]
Hello again @khanitnanj,

Thank you for the CodeSandbox, it helped me debug why you're experiencing this issue 👍.

Please use padding instead of margin for spacing between slides, like [this example setup shows](https://codesandbox.io/s/embla-carousel-react-basic-setup-znjzv). Because Embla won't calculate the margins but the padding will be included when reading the slide widths if you use the border-box property as default like so:

```css
html {
  box-sizing: border-box;
}

*,
*::before,
*::after {
  box-sizing: inherit;
}
```

<img width="1392" alt="embla" src="https://user-images.githubusercontent.com/11529148/83138542-3d094d80-a0eb-11ea-8573-810d7840b456.png">

Let me know if this helps!

Cheers
David

**Comment by davidcetinkaya** [comment]
I'm closing this issue. Feel free to re-open it if you don't think it's resolved.
Thank you @khanitnanj!

**Comment by davidcetinkaya** [comment]
Hi again @khanitnanj,

For what it’s worth, Embla Carousel now supports margin and any other means that will render spacing between your slides. You’re not forced to use the padding approach anymore. And it will automatically detect your spacings! 

This feature was released with version 4.2.0. [Read more about it here](https://www.embla-carousel.com/guides/slide-gaps/) if you’re interested.

Kindly,
David
